app/views/api/v0/articles/show.json.jbuilder

```diff
@@ -1,27 +1,13 @@
-json.type_of                  "article"
-json.id                       @article.id
-json.title                    @article.title
-json.description              @article.description
-json.cover_image              cloud_cover_url @article.main_image
-json.readable_publish_date    @article.readable_publish_date
-json.social_image             article_social_image_url(@article)
-json.tag_list                 @article.cached_tag_list
-json.tags                     @article.cached_tag_list_array
-json.slug                     @article.slug
-json.path                     @article.path
-json.url                      @article.url
-json.canonical_url            @article.processed_canonical_url
-json.comments_count           @article.comments_count
-json.positive_reactions_count @article.positive_reactions_count
-json.collection_id            @article.collection_id
-json.created_at               @article.created_at.utc.iso8601
-json.edited_at                @article.edited_at&.utc&.iso8601
-json.crossposted_at           @article.crossposted_at&.utc&.iso8601
-json.published_at             @article.published_at&.utc&.iso8601
-json.last_comment_at          @article.last_comment_at&.utc&.iso8601
-json.published_timestamp      @article.published_timestamp
+json.partial! "article", article: @article
 
-json.body_html     @article.processed_html
+# /api/articles and /api/articles/:id have opposite representations
+# of `tag_list` and `tags and we can't align them without breaking the API,
+# this is fully documented in the API docs
+# see <https://github.com/thepracticaldev/dev.to/issues/4206> for more details
+json.tag_list @article.cached_tag_list
+json.tags @article.cached_tag_list_array
```

Comment:
added this note because I had to keep these two attributes out of the partial, some of us are familiar with this discrepancy but it's still a good idea to document it